tests/conftest.py

```diff
@@ -111,6 +111,40 @@ def oai_pmh_records():
     return XMLTransformer.parse_source_file("tests/fixtures/oai_pmh_records.xml")
 
 
+# springshare - libguides ##########################
+@pytest.fixture
+def springshare_libguides_record_optional_fields_blank():
+    source_records = OaiDc.parse_source_file(
+        "tests/fixtures/oai_dc/springshare/libguides/libguides_record_optional_fields_blank.xml"
+    )
+    return next(source_records)
+
+
+@pytest.fixture
+def springshare_libguides_record_optional_fields_missing():
+    source_records = OaiDc.parse_source_file(
+        "tests/fixtures/oai_dc/springshare/libguides/libguides_record_optional_fields_missing.xml"
+    )
+    return next(source_records)
+
+
+# springshare - research databases ##########################
+@pytest.fixture
+def springshare_research_databases_record_optional_fields_blank():
+    source_records = OaiDc.parse_source_file(
+        "tests/fixtures/oai_dc/springshare/research_databases/research_databases_record_optional_fields_blank.xml"
+    )
+    return next(source_records)
+
+
+@pytest.fixture
+def springshare_research_databases_record_optional_fields_missing():
+    source_records = OaiDc.parse_source_file(
+        "tests/fixtures/oai_dc/springshare/research_databases/research_databases_record_optional_fields_missing.xml"
+    )
+    return next(source_records)
```

Comment:
As discussed in the `Datacite` PR, we can just use stub records instead of fixtures

Reply:
Ya'll are deeper into these than I am, but agreed.  If we have pre-existing tests that use the full fixtures, seems okay to leave them (?), but any _new_ fixtures or tests ideally would be leaning on the stubs approach.